apps/impala/src/impala/conf.py

```diff
@@ -229,6 +229,13 @@ def get_daemon_api_password():
   dynamic_default=get_daemon_api_username
 )
 
+DAEMON_API_AUTH_SCHEME = Config(
+  key="daemon_api_auth_scheme",
+  help=_t("The authentication scheme to use with 'daemon_api_username' and 'daemon_api_password' when authenticating to the Impala Daemon UI, either 'digest' (default) or 'basic'."),
```

Comment:
As a way of documenting the options, could you also add the info:
# The authentication scheme to use with ....
## daemon_api_auth_scheme=digest

to these files?
https://github.com/cloudera/hue/blob/master/desktop/conf/pseudo-distributed.ini.tmpl#L1355
https://github.com/cloudera/hue/blob/master/desktop/conf.dist/hue.ini#L1370